.github/workflows/tdd-verification.yml

```
@@ -0,0 +1,246 @@
+          modules = [str(m.relative_to("src")).replace(".py", "").replace("/", ".") for m in modules]
+          modules = [m for m in modules if not m.endswith("__init__")]
+
+          # Check for corresponding test files
+          missing_tests = []
+          for module in modules:
+              module_parts = module.split(".")
+              if len(module_parts) > 2:  # Skip __init__ files
+                  module_path = "/".join(module_parts[1:])
+                  test_file = test_dir / f"test_{module_path}.py"
+                  component_test = test_dir / "components" / f"test_{module_parts[-1]}.py"
+                  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)